### ISSUE TITLE
Unable to get files from soxs cache directory

### ISSUE THREAD

**Issue by xshaokun** [lynx-x-ray-observatory/soxs#38]
Since version 4.7.1, the files located in the soxs cache directory cannot be retrieved during instrument simulation and are instead downloaded directly from the database, unless the response files are located in the current directory. This seems different from the previous behavior. I dove into the code and found that the method `get_data_file(fn)` only checks the current directory.

https://github.com/lynx-x-ray-observatory/soxs/blob/57c3a23c932296cdfadf06a571dd06fde4cfc030/soxs/utils.py#L308C1-L317C1

**Comment by xshaokun** [comment]
###  More Details

I create a new instrument specification as follows
```python
chandra_circinus = soxs.get_instrument_from_registry("chandra_aciss_cy0")
chandra_circinus["name"] = "chandra_aciss_circinus"
chandra_circinus["arf"] = "Circinus_W_lobe_comb_src.arf"
chandra_circinus["rmf"] = "Circinus_W_lobe_comb_src.rmf"
name = soxs.add_instrument_to_registry(chandra_circinus)
```
If the file `Circinus_W_lobe_comb_src.arf` is located in the soxs cache directory instead of the current directory, the instrument simulation raises a `ValueError`

```python
ValueError: File 'Circinus_W_lobe_comb_src.arf' is not in the registry.
```

**Comment by jzuhone** [comment]
This should be fixed in 4696f9b7c5b4f652544a0598632e2d0e702df8dd, please let me know if it's not.

**Comment by xshaokun** [comment]
It works. Thanks! @jzuhone
